### ISSUE TITLE
I am reproducing your model result, and I have a question about the code

### ISSUE THREAD

**Issue by shoutOutYangJie** [THU-MIG/YOLO-UniOW#18]
In the "YOLOv10WorldHead", you use ".detach()" to cut the grad from "one2one" supervise loss. I don't know why

![Image](https://github.com/user-attachments/assets/074586d4-565b-49dd-870c-73b617eea133)
